docs/server-side-website.md

```diff
@@ -346,6 +346,30 @@ constructs:
 
 CloudFront accepts maximum 10 headers.
 
+## Extensions
+
+You can specify an `extensions` property on the `server-side-website` construct to extend the underlying CloudFormation resources. In the exemple below, the CloudFront Distribution CloudFormation resource generated by the `website` server-side-website construct will be extended with the new `Comment: Landing distribution` CloudFormation property.
+
+```yaml
+constructs:
+    website:
+        type: server-side-website
+        assets:
+            '/css/*': public/css
+            '/js/*': public/js
+        extensions:
+            distribution:
+                Properties:
+                    Comment: Landing distribution
+```
+
+### Available extensions
+
+| Extension key 	| CloudFormation resource       	| CloudFormation documentation                                                                                           	|
+|---------------	|-------------------------------	|------------------------------------------------------------------------------------------------------------------------	|
+| distribution  	| AWS::CloudFront::Distribution 	| [Link](https://docs.aws.amazon.com/fr_fr/AWSCloudFormation/latest/UserGuide/aws-resource-cloudfront-distribution.html) 	|
```

Comment:
hello,

may I ask a question, more than a comment?
For this type of builder, we will also create an S3 bucket, will it be possible to adjust it, like the "storage" builder, or will we only be able to adjust the distribution?

Reply:
Absolutely possible :) I'll merge this one, if you want to provide a PR on the basis of this new code base to implement the storage extension on this construct, you're welcome to :)

Reply:
I would love to, but I can't compile the project properly, nor can I run the unit tests without errors on my side (on the branch master). So it's hard to try implementations :/